packages/client/modules/userDashboard/components/OrgAuthentication/OrgAuthentication.tsx

```diff
@@ -31,18 +40,19 @@ const OrgAuthentication = (props: Props) => {
     queryRef
   )
   const {viewer} = data
-  const {organization} = viewer
+  const {organization, featureFlags} = viewer
+  const {checkoutFlow} = featureFlags
   const saml = organization?.saml ?? null
   const disabled = !saml
   return (
-    <Panel>
+    <StyledPanel isWide={checkoutFlow}>
```

Comment:
Normally we should adjust the design based on the parents, but as we're going to remove this flag soon, I just copied what we do in other places already.